comprobo_final_project/scripts/models/robot.py

```diff
@@ -1,2 +1,44 @@
+#!usr/bin/env python
+
+import rospy
+from geometry_msgs.msg import PoseStamped, Twist
+from ..providers.gazebo_position_provider import GazeboPoseProvider
+
 class Robot:
-    pass
+    """
+    Neato connection.
+    set_twist(twist : Twist) : Void - Sets the Twist of the robot
+    """
+
+    def __init__(self, position_listener):
```

Comment:
i'm fairly confused by the interactions between this file and the other. I will currently just assume that they work.